src/foundry/common/documents/user.d.mts

```diff
@@ -374,4 +353,6 @@ declare namespace BaseUser {
    * @deprecated {@link BaseUser.CreateData | `BaseUser.CreateData`}
    */
   type ConstructorData = BaseUser.CreateData;
+
+  type TestablePermissionOrRole = keyof typeof CONST.USER_PERMISSIONS | CONST.USER_ROLE_NAMES | CONST.USER_ROLES;
```

Comment:
So we try to put helper types in `User` and re-export in `BaseUser`. It's easier to write `User.XYZ` than `foundry.documents.BaseUser.XYZ`.